### PR TITLE
airtable: Include phrase describing the status code in error

### PR DESCRIPTION
**Pull request by jcharum**

### CODE REVIEW

File: airtable/plugin.py [hunk 1]
```diff
@@ -19,13 +19,22 @@ def _resolve_error_message(text: str) -> str:
         data = json.loads(text)
     except json.JSONDecodeError:
         return text
```
Comment (marcrasi) [inline airtable/plugin.py:21]: do you also want to include the code and phrase in this case?

File: airtable/plugin.py [hunk 2]
```diff
@@ -19,13 +19,22 @@ def _resolve_error_message(text: str) -> str:
...
+    match (error := data.get("error")):
+        case str():
+            return (
+                f"{status_code} {httpx.codes.get_reason_phrase(status_code)}: {error}"
+            )
+        case dict():
+            error_type = error.get("type")
+            error_message = error.get("message")
+            if not isinstance(error_type, str) or not isinstance(error_message, str):
+                return text
```
Comment (marcrasi) [inline airtable/plugin.py:31]: and this one?

File: airtable/plugin.py [hunk 3]
```diff
@@ -19,13 +19,22 @@ def _resolve_error_message(text: str) -> str:
...
+            return (
+                f"{status_code} {httpx.codes.get_reason_phrase(status_code)}: "
+                f"{error_type}: {error_message}"
+            )
+        case _:
+            return text
```
Comment (marcrasi) [inline airtable/plugin.py:37]: and this one?

**Review by marcrasi — APPROVED** [review]

File: airtable/plugin.py (hunk 2 above)
Comment (jcharum) [reply airtable/plugin.py]: Yes.

File: airtable/plugin.py (hunk 3 above)
Comment (jcharum) [reply airtable/plugin.py]: Yes.

**Pull request merged**